grails-app/conf/UrlMappings.groovy

```diff
@@ -425,6 +426,7 @@ class UrlMappings {
         name politicianMassMailingRemove: "/ajax/account/mass-mailing/$campaignId/remove"(controller: "newsletter", action: "removeCampaign")
         name politicianMassMailingTrackEvents: "/ajax/account/mass-mailing/$campaignId/trackEvents"(controller: "newsletter", action: "showTrackingMails")
         name politicianMassMailingTrackEventsResend: "/ajax/account/mass-mailing/$campaignId/trackEvents/resend/$tackingMailId"(controller: "newsletter", action: "resendEmail")
+        name politicianMassMailingBulletinResend: "/ajax/account/mass-mailing/$campaignId/resend/$contactId"(controller: "newsletter", action: "copyBulletinAndSend")
```

Comment:
Is the same as API. Is it a "resend"?

Reply:
Changed to 

`politicianMassMailingBulletinCopyAndSend: "/ajax/account/mass-mailing/$campaignId/send/$contactId"(controller: "newsletter", action: "copyBulletinAndSend")`